<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Anansi::FileAssociationsWidget</class>
 <widget class="QWidget" name="Anansi::FileAssociationsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTreeView" name="fileExtensionMediaTypes">
     <property name="animated">
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="mediaTypeButtonsLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addEntry">
       <property name="toolTip">
        <string>&lt;p&gt;Add an association between a file name extension and a media type.&lt;/p&gt;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="list-add" resource="../resources/resources.qrc">
         <normaloff>:/icons/buttons/add-to-list</normaloff>:/icons/buttons/add-to-list</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeEntry">
       <property name="toolTip">
        <string>&lt;p&gt;Remove the selected file name extension or media type.&lt;/p&gt;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="list-remove" resource="../resources/resources.qrc">
         <normaloff>:/icons/buttons/remove-from-list</normaloff>:/icons/buttons/remove-from-list</iconset>
       </property>
       <property name="autoRepeatDelay">
        <number>317</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="defaultMediaTypeLayout" stretch="0,1">
     <item>
      <widget class="QLabel" name="defaultMediaTypeLabel">
       <property name="text">
        <string>Default media Type</string>
       </property>
       <property name="buddy">
        <cstring>defaultMediaType</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Anansi::MediaTypeCombo" name="defaultMediaType"/>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionAddExtension">
   <property name="text">
    <string>Add extension</string>
   </property>
   <property name="toolTip">
    <string>&lt;p&gt;Add a new filename extension to the list.&lt;/p&gt;
&lt;p&gt;&lt;small&gt;Adding a filename extension enables the web server to recognise files with that extension.&lt;/small&gt;&lt;/p&gt;</string>
   </property>
  </action>
  <action name="actionAddMediaType">
   <property name="text">
    <string>Add media type</string>
   </property>
   <property name="toolTip">
    <string>&lt;p&gt;Add a media type to the current file extension.&lt;/p&gt;
&lt;p&gt;&lt;small&gt;Adding a media type enables files with the associated extension to be handled by the web server as that media type.&lt;/small&gt;&lt;/p&gt;</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Anansi::MediaTypeCombo</class>
   <extends>QComboBox</extends>
   <header>src/mediatypecombo.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>fileExtensionMediaTypes</tabstop>
  <tabstop>addEntry</tabstop>
  <tabstop>removeEntry</tabstop>
  <tabstop>defaultMediaType</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
